python/lsst/afw/geom/utils.py

```diff
@@ -25,19 +24,23 @@
 
 In the case of the assert functions, importing them makes them available in lsst.utils.tests.TestCase
 """
+__all__ = ["assertAnglesAlmostEqual", "assertPairsAlmostEqual",
```

Comment:
Yeah, Russell and I discussed this, I was going on the dev guide which had not been updated at the point of me reviewing this. He has now updated the dev guide